fixtures/User.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Nelmio\Alice;
+
+/**
+ * @package Nelmio\Alice
+ */
+class User
+{
+    private $name;
+
+    /**
+     * @return mixed
+     */
+    public function getName()
```

Comment:
we're in PHP 7.1 so we can add the return type `?string` and remove the redundant/useless phpdoc